package.json

```diff
@@ -61,12 +57,10 @@
     "server:silent": "python network-api/manage.py runserver 0.0.0.0:8000 >> server.log 2>&1",
     "start": "docker-compose up",
     "sync": "browser-sync start --proxy \"localhost:8000\" --reload-debounce 50 --files \"./network-api/networkapi/**/*.html\" \"./network-api/networkapi/frontend/_css/*.css\" \"./network-api/networkapi/frontend/_js/*.js\"",
-    "tailwind": "run-p tailwind:watch sync",
-    "tailwind:watch": "npm run build:tailwind -- -w",
     "test": "npm run lint",
-    "watch": "wait-on http://backend:8000/cms && run-s build:clean && run-p build:js:dev build:common watch:**",
+    "watch": "wait-on http://backend:8000/cms && run-s build:clean && run-p build:common && run-p build:js:dev postcss:watch watch:**",
     "watch:images": "chokidar \"source/images/**/*\" -c \"npm run build:images\"",
-    "watch:sass": "chokidar \"source/**/*.scss\" \"network-api/**/*.html\" -c \"npm run build:sass\""
+    "watch:scss": "chokidar \"source/**/*.scss\" -c \"npm run build:css\""
```

Comment:
We shouldn't need to watch *.html file changes to trigger `build:sass`. postcss's watch mode along with tailwindcss plugin will do the job already. (See `postcss:watch` task)